frontend_tests/puppeteer_tests/recent-topic.ts

```diff
@@ -233,6 +233,37 @@ async function test_all_filters(page: Page): Promise<void> {
     await trigger_filter_button(page, participated_filter_button);
 }
 
+// Tests the `t` hotkey which navigates back to recent topic page.
+async function test_recent_topic_hotkey(page: Page): Promise<void> {
+    await page.waitForSelector("#zfilt", {visible: true});
+    await page.keyboard.press("t");
+    await page.waitForSelector(recent_topic_page, {visible: true});
+}
+
+async function test_stream_navigation(page: Page, stream_name: string): Promise<void> {
+    await page.waitForSelector(recent_topic_page, {visible: true});
+
```

Comment:
I don't think so we need this function.

As we are using it only once.

You can study this function about how to navigate with the stream, https://github.com/zulip/zulip/blob/bf056c899032d474d1a2f1546d15ff00ea264afd/frontend_tests/puppeteer_tests/navigation.ts#L12